R/format_ae_specific.R

```diff
@@ -68,30 +68,23 @@ format_ae_specific <- function(outdata,
 
   # Define total column
   display_total <- "total" %in% display
-  index_total <- ncol(outdata$n)
+  index_total <- if (display_total) {1:ncol(outdata$n)} else {1:(ncol(outdata$n)-1)}
```

Comment:
Try to avoid `if else` statement. 

```suggestion
  index_total <- seq(ncol(outdata$n) - (! display_total) )
```